src/functions/getTestCentres/framework/repositories/active-test-centres.ts

```diff
@@ -0,0 +1,15 @@
+import * as mysql from 'mysql2';
+import { info } from '@dvsa/mes-microservice-common/application/utils/logger';
+import { getTestCentres } from '../database/query-builder';
+import { getConnection } from '../../../../common/config/connection';
+import { query } from '../../../../common/framework/mysql/database';
+
+/**
+ * Call TARS replica for a list of all test centres
+ */
+export const findTestCentres = async (): Promise<any> => {
+  const connection: mysql.Connection = getConnection();
+
+  info('Searching for all test centres');
+  return query(connection, await getTestCentres());
```

Comment:
In other get instances, we also run connection.end such as here
```
export const getResult = async (
  appRef: number,
): Promise<TestResultRecord[]> => {
  const connection: mysql.Connection = getConnection();
  let batch;
  try {
    const [rows] = await connection.promise().query(
      buildGetResultQuery(appRef),
    );
    batch = rows;
  } finally {
    connection.end();
  }
  return batch;
};
```
Should we be doing same here?

Reply:
sure, not there tho.  as a common funtion. dont want that repeated for each service